serverless.json

```diff
@@ -31,6 +31,12 @@
     "region": "eu-central-1",
     "deploymentBucket": {
       "name": "kaskadi-serverless-deployment-bucket"
+    },
+    "tags": {
+      "app": "your-app-name",
+      "service": "${self:service.name}",
+      "logical-unit": "api-logical-unit",
+      "type": "api, s3, sns, etc."
```

Comment:
I implemented tags at API level following [this issue](https://github.com/kaskadi/template-kaskadi-api/issues/11#issue-578768729).

- `app` is our encapsulating app
- `service` no need to modify manually, this is our service
- `logical-unit` is the logical unit for our API like `stocks`, `auth`, `webhooks`, etc.
- `type` is the type of lambda we have here. Normally only `api` but maybe some are triggered by something else also and we want to tag it so we can find it again later on

Reply:
i'm not sure about the names but otherwise looks nice. as i have no better idea for naming, we go with this for now...